client/components/Treemap.jsx

```diff
@@ -105,9 +105,12 @@ export default class Treemap extends Component {
       },
       onGroupDoubleClick: preventDefault,
       onGroupHover(event) {
-        // Ignoring hovering on `FoamTree` branding group
-        if (event.group && event.group.attribution) {
+        // Ignoring hovering on `FoamTree` branding group and the root group
+        if (event.group && (event.group.attribution || event.group === this.get('dataObject'))) {
           event.preventDefault();
+          if (props.onMouseLeave) {
```

Comment:
Do we need this conditional? Is there a place in the code where `props.onMouseLeave` would not be specified?

Reply:
I made the condition analogous to the `props.onGroupHover` check below, which is also there but probably not required:

```
        if (props.onGroupHover) {
          props.onGroupHover.call(component, event);
        }
```

I can remove the `props.onMouseLeave` condition, not a problem.